source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+---
+title: "Customize the S3-BOX with your own images"
+product_name: ESP32-S3-BOX
+device_name_entry: ESP32-S3-BOX
+config_link: /voice_control/thirteen-usd-voice-remote/#to-delete-the-atom-echo-configuration-from-esphome
+---
+
+This tutorial will show you how to replace the Home Assistant status images on the ESP32-S3-BOX by custom images. 
+
+You can either prepare your own images or use images from a community repository.
+It applies to the following three ESP32 devices by Espressif:
+
+- [ESP32-S3-BOX](https://www.espressif.com/en/news/ESP32-S3-BOX_video)
+- [ESP32-S3-BOX-3](https://www.espressif.com/en/news/ESP32-S3-BOX-3)
+- [ESP32-S3-BOX-Lite](https://www.espressif.com/en/news/ESP32-S3-BOX_video)
+
+The steps are the same for all three product variants. The term *ESP32-S3-BOX* is used to refer to all three variants.
+
+## Prerequisites
+
+- Latest version of Home Assistant, installed with the Home Assistant Operating System
+- [Home Assistant Cloud](https://www.nabucasa.com) or a manually configured [Assist Pipeline](/voice_control/voice_remote_local_assistant)
+- Successfully completed the [ESP32-S3-BOX voice assistant](/voice_control/s3_box_voice_assistant/) tutorial
+
+## Adopting the device in the ESPHome add-on
+
+Before you can import new images, you need to install the ESPHome add-on and adopt the device in the add-on.
+
+1. Make sure the ESP32-S3-BOX is up and running and connected to your Wi-Fi.
+2. Go to **Settings** > **Add-ons** and check if you have the **ESPHome** add-on installed.
+   - If you haven't done so, install the add-on.
```

Comment:
Add a My link to link/install the add-on

Reply:
```
[![Open your Home Assistant instance and show the dashboard of an add-on.](https://my.home-assistant.io/badges/supervisor_addon.svg)](https://my.home-assistant.io/redirect/supervisor_addon/?addon=5c53de3b_esphome)
```